### PR TITLE
FIX example jobs so that ..

### PR DESCRIPTION
**Pull request by imuchnik**
We have working software OVER Docs
but docs are there too.

### CODE REVIEW

**Comment by imuchnik** [comment]
@marcesher @sleitner @contolini @vickumar1981 @OrlandoSoto  Please review

File: docs/examples.md [hunk 1]
```diff
@@ -63,13 +64,13 @@ import jenkins.automation.builders.BaseJobBuilder
 ```
 import jenkins.automation.builders.FlowJobBuilder
 
-    def oahMaster= new FlowJobBuilder(
+    def flowExampleJob= new FlowJobBuilder(
             name: 'GeneratedFlowJob',
             description: 'this our first stab at it',
             jobs:['job1', 'job2']
```
Comment (sleitner) [inline docs/examples.md:70]: @vickumar1981 added a jobFlow string variable for Flows that include more than 'build(job)' syntax, so we should mention that here.

Comment (imuchnik) [inline docs/examples.md:70]: Excellent point @sleitner. @vickumar1981  please add an example. :smile:

Comment (vickumar1981) [inline docs/examples.md:70]: @sleitner @marcesher  Done.

https://github.com/cfpb/jenkins-automation/pull/33

**Comment by vickumar1981** [comment]
@imuchnik  needs to be rebased :)

**Pull request merged**